### ISSUE TITLE
VRP - "A fatal error has been detected by the Java Runtime Environment"

### ISSUE THREAD

**Issue by cuongsimplypost** [google/or-tools#691]
Hi All,
I try to run vrp example in java which ported from [google guides](https://developers.google.com/optimization/routing/vrp )
I got this runtime error: 
`A fatal error has been detected by the Java Runtime Environment:
 SIGSEGV (0xb) at pc=0x000000013603340d, pid=7709, tid=36611
JRE version: Java(TM) SE Runtime Environment (10.0.1+10) (build 10.0.1+10)
Java VM: Java HotSpot(TM) 64-Bit Server VM (10.0.1+10, mixed mode, tiered, compressed oops, g1 gc, bsd-amd64)
Problematic frame:
C  [libortools.dylib+0x2c740d]  operations_research::Assignment::Value(operations_research::IntVar const*) const+0xd`

Here is full log: [hs_err_pid7709.log](https://github.com/google/or-tools/files/2003347/hs_err_pid7709.log)

Here is my code: 
`public class VehicleRoutingProblem {

    static {
        System.load(System.getProperty("user.dir") + "/libs/libjniortools.jnilib");
    }

    private static Logger logger =
            Logger.getLogger(VehicleRoutingProblem.class.getName());

    // Locations representing either an order location or a vehicle route
    // start/end.
    private List<Pair<Integer, Integer>> locations = new ArrayList();

    // Capacity of the vehicles.
    private int numberOfVehicles = 4;

    public void solve() {
        locations.add(Pair.of(4, 4)); //Depot
        locations.add(Pair.of(2, 0));
        locations.add(Pair.of(8, 0));
        locations.add(Pair.of(0, 1));
        locations.add(Pair.of(1, 1));
        locations.add(Pair.of(5, 2));
        locations.add(Pair.of(7, 2));
        locations.add(Pair.of(3, 3));
        locations.add(Pair.of(6, 3));
        locations.add(Pair.of(5, 5));
        locations.add(Pair.of(8, 5));
        locations.add(Pair.of(1, 6));
        locations.add(Pair.of(2, 6));
        locations.add(Pair.of(3, 7));
        locations.add(Pair.of(6, 7));
        locations.add(Pair.of(0, 8));
        locations.add(Pair.of(7, 8));

        final int numberOfLocations = locations.size();
        final int numberOfOrders = numberOfLocations - 1;

        logger.info("Creating model with " + numberOfOrders + " orders and " +
                numberOfVehicles + " vehicles.");

        // Finalizing model

        RoutingModel model = new RoutingModel(numberOfLocations, numberOfVehicles, 0);

        // Setting up dimensions
        NodeEvaluator2 manhattanDistance = new NodeEvaluator2(){
            @Override
            public long run(int firstIndex, int secondIndex) {
                try {
                    Pair<Integer, Integer> firstLocation = locations.get(firstIndex);
                    Pair<Integer, Integer> secondLocation = locations.get(secondIndex);
                    return Math.abs(firstLocation.first - secondLocation.first) +
                            Math.abs(firstLocation.second - secondLocation.second);
                } catch (Throwable throwed) {
                    logger.warning(throwed.getMessage());
                    return 0;
                }
            }
        };
        model.setArcCostEvaluatorOfAllVehicles(manhattanDistance);

        // Solving
        RoutingSearchParameters parameters =
                RoutingSearchParameters.newBuilder()
                        .mergeFrom(RoutingModel.defaultSearchParameters())
                        .setFirstSolutionStrategy(FirstSolutionStrategy.Value.PATH_CHEAPEST_ARC)
                        .build();

        logger.info("Search");
        Assignment solution = model.solveWithParameters(parameters);

        if (solution != null) {
            String output = "Total cost: " + solution.objectiveValue() + "\n";
            // Dropped orders
            String dropped = "";
            for (int order = 0; order < numberOfOrders; ++order) {
                if (solution.value(model.nextVar(order)) == order) {
                    dropped += " " + order;
                }
            }
            if (dropped.length() > 0) {
                output += "Dropped orders:" + dropped + "\n";
            }
            // Routes
            for (int vehicle = 0; vehicle < numberOfVehicles; ++vehicle) {
                String route = "Vehicle " + vehicle + ": ";
                long order = model.start(vehicle);
                if (model.isEnd(solution.value(model.nextVar(order)))) {
                    route += "Empty";
                } else {
                    for (;
                         !model.isEnd(order);
                         order = solution.value(model.nextVar(order))) {
                        IntVar load = model.cumulVar(order, "capacity");
                        IntVar time = model.cumulVar(order, "time");
                        route += order + " Load(" + solution.value(load) + ") " +
                                "Time(" + solution.min(time) + ", " + solution.max(time) +
                                ") -> ";
                    }
                    IntVar load = model.cumulVar(order, "capacity");
                    IntVar time = model.cumulVar(order, "time");
                    route += order + " Load(" + solution.value(load) + ") " +
                            "Time(" + solution.min(time) + ", " + solution.max(time) + ")";
                }
                output += route + "\n";
            }
            logger.info(output);
        }
    }
}`

**Comment by Mizux** [comment]
Hi,
How did you get your or-tools version ? Did you rebuild it from source or use a binary tar ball ?
Also did you manage to run this sample https://github.com/google/or-tools/blob/master/examples/com/google/ortools/samples/Tsp.java ?

**Comment by cuongsimplypost** [comment]
Hi Mizux

I get or-tools from [google installing binary guides](https://developers.google.com/optimization/introduction/installing/binary) (binary package)
I use the binary(.jars) lib from that package.

I haven't run the Tsp.java example but I can run the CapacitatedVehicleRoutingProblemWithTimeWindows successfully.

**Comment by cuongsimplypost** [comment]
Hi
I just found the reason. This is an error when I try to log "time" while I don't setup any time constrain here.
Please help to close the issue

**Comment by Mizux** [comment]
Happy that you fix (yourself) your issue !

So you mean `  IntVar time = model.cumulVar(order, "time");` throw without any relevant message ?
e.g. `No dimension "time" found`

**Comment by cuongsimplypost** [comment]
@Mizux : Yup, it crashed when hit the line `IntVar time = model.cumulVar(order, "time");`
